src/components/UI/Text/InputNumber.vue

```diff
@@ -257,7 +262,11 @@ export default {
     },
     keyPress(event) {
       // invoked when a key is pressed
-      if (this.maxLength != null && this.localValue.length == this.maxLength) {
+      // character key not accepted
+      if (!/^[0-9]+$/.test(event.key)){
+        event.preventDefault();
+      }
```

Comment:
You can do this in the <input> tag itself. No need to write extra javascript for this
Take a look at this 
https://www.w3schools.com/tags/att_input_pattern.asp

Reply:
yes I tried this earlier too it was not working for me

Reply:
Okay no issues then